sdk/servicebus/service-bus/test/public/sessionsTests.spec.ts

```diff
@@ -300,6 +300,25 @@ describe("session tests", () => {
         err.name.should.equal("AbortError");
       }
     });
+
+    ["acceptSession", "acceptNextSession"].forEach((method) => {
+      it(`${method} on session id of empty string`, async () => {
+        serviceBusClient = createServiceBusClientForTests();
+        const entityNames = await serviceBusClient.test.createTestEntities(testClientType);
+        const testMessage = TestMessage.getSessionSample();
+        testMessage.sessionId = "";
+        sender = serviceBusClient.test.addToCleanup(
+          serviceBusClient.createSender(entityNames.queue ?? entityNames.topic!),
+        );
+        await sender.sendMessages(testMessage);
+        receiver =
+          method === "acceptSession"
+            ? await serviceBusClient.test.acceptSessionWithPeekLock(entityNames, "")
+            : await serviceBusClient.test.acceptNextSessionWithPeekLock(entityNames);
```

Comment:
nitpick:
```suggestion
          await method === "acceptSession"
            ? serviceBusClient.test.acceptSessionWithPeekLock(entityNames, "")
            : serviceBusClient.test.acceptNextSessionWithPeekLock(entityNames);
```

Reply:
Hmm, I prefer to have `await` closer the promise/async method. Let me know if you feel strong about it.